Makefile

```diff
@@ -85,7 +85,7 @@ opencv-cp-arm64:
 	savant-ds-opencv-l4t:$(DEEPSTREAM_VERSION)
 
 run-docs:
-	docker run -it --rm \
+	docker run -it --rm --gpus all \
```

Comment:
Yes, otherwise the error when generating autosammary. 

terminate called after throwing an instance of 'cv::Exception'
  what():  OpenCV(4.6.0) /opencv/opencv/modules/core/src/cuda_stream.cpp:305: error: (-217:Gpu API call) CUDA driver version is insufficient for CUDA runtime version in function 'Impl'